truss-chains/examples/transcribe/whisper_model.py

```diff
@@ -34,7 +32,7 @@ class WhisperModel(chains.ChainletBase):
             ],
             external_package_dirs=[chains.make_abs_path_here(_LOCAL_WHISPER_LIB)],
         ),
-        compute=chains.Compute(gpu="A10G", predict_concurrency=128),
+        compute=chains.Compute(gpu="H100", predict_concurrency=128),
```

Comment:
For exactly cases like this a feature like a file-based deployment override would be cool at some point (like ray serve has).